script.js

```diff
@@ -0,0 +1,21 @@
+function Newbook() {
+  const Title = document.getElementById('Title');
+  const Author = document.getElementById('Author');
+  const newdiv = document.createElement('div');
+  newdiv.setAttribute('id', 'book_infos');
+  const par1 = document.createElement('p');
+  const par2 = document.createElement('p');
+  const buutt = document.createElement('button');
+  buutt.textContent = 'remove';
+  buutt.setAttribute('id', 'rem');
+  par1.appendChild(document.createTextNode(Title.value));
+  par2.appendChild(document.createTextNode(Author.value));
+  document.getElementById('list_container').appendChild(newdiv).appendChild(par1);
+  document.getElementById('list_container').appendChild(newdiv).appendChild(par2);
+  document.getElementById('list_container').appendChild(newdiv).appendChild(buutt);
+}
```

Comment:
- [ ] [OPTIONAL] Break this function in two or three functions, or just one but without all the get elements, remember than a function must do one thing and one thing only, e.g. you can take out all the get elements on a global scope and let the function to just add the newbook.